doc/classes/SpinBox.xml

```diff
@@ -13,23 +26,29 @@
 			<return type="LineEdit">
 			</return>
 			<description>
+				Returns the [LineEdit] id of the [SpinBox].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="align" type="int" setter="set_align" getter="get_align" enum="LineEdit.Align">
+			Sets the text alignment of the [SpinBox].
 		</member>
 		<member name="editable" type="bool" setter="set_editable" getter="is_editable">
+			If [code]true[/code], the [SpinBox] will be editable. Otherwise, it will not. Default value is [code]true[/code]
 		</member>
 		<member name="prefix" type="String" setter="set_prefix" getter="get_prefix">
+			Sets a prefix for the [SpinBox].
```

Comment:
It would be good to be more explicit on what a prefix and a suffix mean in the content of SpinBox.